lib/Models/CkanCatalogGroup.js

```diff
@@ -329,8 +329,7 @@ CkanCatalogGroup.prototype._load = function() {
 
     var promises = [];
     for (var i = 0; i < this.filterQuery.length; i++) {
-        var url = cleanAndProxyUrl(this, this.url) + '/api/3/action/package_search?rows=100000&' + this.filterQuery[i];
-
+        var url = cleanAndProxyUrl( this.terria, this.url) + 'api/3/action/package_search?rows=100000&' + this.filterQuery[i];
```

Comment:
Is this a bad merge?  I believe the original code is correct and the new code is wrong.